frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -295,9 +308,63 @@ class Chart extends Component<Props, State> {
         });
     }
 
+    private handleCrosshair = ({ point, time, seriesPrices }) => {
+        if (!this.refToolTip) {
+            return;
+        }
+        const tooltip = this.refToolTip.current;
+        const parent = tooltip.parentNode;
+        if (
+            !this.lineSeries || !point || !time
+            || point.x < 0 || point.x > parent.clientWidth
+            || point.y < 0 || point.y > parent.clientHeight
+        ) {
+            this.setState({
+                toolTipVisible: false
+            });
+            return;
+        }
+        const price = seriesPrices.get(this.lineSeries);
+        const coordinate = this.lineSeries.priceToCoordinate(price);
+        if (!coordinate) {
+            return;
+        }
+        const toolTipTop = Math.max(coordinate - 70, 0);
+        const toolTipLeft =  Math.max(40, Math.min(parent.clientWidth - 160, point.x + 40 - 80));
+        this.setState({
+            toolTipVisible: true,
+            toolTipValue: price,
+            toolTipTop,
+            toolTipLeft,
+            toolTipTime: time,
+        });
+    }
+
+    private renderDate = (date) => {
+        date = new Date(date);
```

Comment:
check if you can use toLocaleString() to format in the same way as the date formatting in the transactions list:

https://github.com/digitalbitbox/bitbox-wallet-app/blob/cc77c1381617a2abb83c4897fa8c0997bc0b6c59/frontends/web/src/components/transactions/transaction.tsx#L81

Reply:
fixed but forced to always display 2 digits

Reply:
LGTM. you are not showing the hour any more in weekly, in case you dropped it by accident. I am okay with either. Feel free to merge.

Reply:
yes I forgot but @jadzeidan didn't like it anyways… waiting for his input.